funcbench/Dockerfile

```diff
@@ -1,21 +1,16 @@
-FROM golang:alpine AS builder
+FROM golang:1.13-alpine AS builder
 
-RUN apk add git
-
-ENV GO111MODULE=on
+WORKDIR /src
 
-WORKDIR /
-COPY . /
+COPY funcbench/*.go ./
+COPY go.* ./
 
-RUN CGO_ENABLED=0 go build main.go
+RUN CGO_ENABLED=0 GO111MODULE=on go build -a -tags netgo -o /go/bin/funcbench
 
-FROM golang:alpine
+FROM golang:1.13-alpine
```

Comment:
`quay.io/prometheus/busybox:latest`?

Reply:
ok  scratch that, I missed that you need `git`.

Reply:
Yes, this is another place we can improve, e.g. use `go-git` to save `git` cli.